docs/content/docs/build/tokenproviders.md

```diff
@@ -4,149 +4,8 @@ menuPosition: 3
-
-```typescript
-import { AzureFunction, Context, HttpRequest } from "@azure/functions";
-import { ScopeType } from "@fluidframework/azure-client";
-import { generateToken } from "@fluidframework/azure-service-utils";
-
-//Replace "myTenantKey" with your key here.
-const key = "myTenantKey";
-
-const httpTrigger: AzureFunction = async function (
-    context: Context,
-    req: HttpRequest
-): Promise<void> {
-    // tenantId, userId and userName are required parameters. documentId is only required for fetching existing containers
-    const tenantId = (req.query.tenantId ||
-        (req.body && req.body.tenantId)) as string;
-    const documentId = (req.query.documentId ||
-        (req.body && req.body.documentId)) as string;
-    const userId = (req.query.userId || (req.body && req.body.userId)) as string;
-    const userName = (req.query.userName ||
-        (req.body && req.body.userName)) as string;
-    const additionalDetails =
-        req.query.additionalDetails || (req.body && req.body.additionalDetails);
-    const scopes = (req.query.scopes ||
-        (req.body && req.body.scopes)) as ScopeType[];
-
-    if (!tenantId) {
-        context.res = {
-            status: 400,
-            body: "No tenantId provided in query params",
-        };
-    }
-
-    if (!key) {
-        context.res = {
-            status: 404,
-            body: `No key found for the provided tenantId: ${tenantId}`,
-        };
-    }
-
-    let user = {
-        name: userName,
-        id: userId,
-        additionalDetails: additionalDetails ? JSON.parse(additionalDetails) : {},
-    };
-
-    // Will generate the token and returned by an ITokenProvider implementation to use with the AzureClient.
-    const token = generateToken(
-        tenantId,
-        documentId,
-        key,
-        scopes ?? [ScopeType.DocRead, ScopeType.DocWrite, ScopeType.SummaryWrite],
-        user
-    );
-
-    context.res = {
-        status: 200,
-        body: token,
-    };
-};
-
-export default httpTrigger;
-```
-
-The `generateToken` function will generate a token for the given user that is signed using the tenant's secret key. This allows the token to be returned to the client without ever exposing the secret itself to it. Instead, the token is generated using it to provide scoped access to the given document. This token can be returned by an ITokenProvider implementation to use with the AzureClient.
-
-## Adding custom data to tokens
-
-You can add custom data such as email id, gender, address, etc for your token generation. See `Token Provider` under [Connect to Azure Fluid Relay]({{< relref "azure-frs.md" >}}) for more information.
+To learn more about using TokenProviders, see [How to: Write a TokenProvider with an Azure Function](https://learn.microsoft.com/en-us/azure/azure-fluid-relay/how-tos/azure-function-token-provider).
```

Comment:
```suggestion
To learn more about using TokenProviders, see [How to: Write a TokenProvider with an Azure Function](https://learn.microsoft.com/azure/azure-fluid-relay/how-tos/azure-function-token-provider).
```

If you remove the en-us from the URL, it'll redirect to localized docs if available.